services/minespace-web/common/actionCreators/minespaceActionCreator.js

```diff
@@ -60,13 +60,20 @@ export const fetchMinespaceUserMines = (mine_guids) => (dispatch) => {
   dispatch(showLoading());
   dispatch(request(reducerTypes.GET_MINESPACE_USER_MINES));
   return CustomAxios()
-    .post(ENVIRONMENT.apiUrl + API.MINE_BASIC_INFO_LIST, { mine_guids }, createRequestHeader())
+    .post(
+      ENVIRONMENT.apiUrl + API.MINE_BASIC_INFO_LIST,
+      { mine_guids, simple: true },
```

Comment:
shouldn't this be parameterized?

Reply:
or perhaps be removed?

Reply:
I don't really see why.  We were already passing the mine_guids as part of the payload in a post, so adding the boolean here makes sense to me.  If we removed it, it wouldn't send the simple flag at all.

Reply:
FYI This isn't actually being used in minespace anywhere.  It's just duplicated for the sake of the common folders.

Reply:
I thought minespace need the everything related to a mine...

Reply:
No that was a different function call (within Core)